hazelcast/src/main/java/com/hazelcast/cp/internal/datastructures/countdownlatch/proxy/CountDownLatchProxy.java

```diff
@@ -76,6 +77,8 @@ public int getCount() {
 
     @Override
     public boolean trySetCount(int count) {
+        checkPositive(count, "Count must be positive!");
```

Comment:
positive or non-negative?

Reply:
The message is correct, it must be positive, i.e. zero is not allowed. It was the behavior without this local check.